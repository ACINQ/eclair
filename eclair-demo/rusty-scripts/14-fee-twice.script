# We can change fee level twice.
A:feechange
A:feechange
A:commit
B:recvfeechange
B:recvfeechange
B:recvcommit
B:commit
A:recvrevoke
A:recvcommit
B:recvrevoke
echo ***A***
A:dump
echo ***B***
B:dump
