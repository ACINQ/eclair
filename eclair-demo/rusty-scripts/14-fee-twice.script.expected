***A***
LOCAL COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs:
  Fee level 2
  SIGNED
REMOTE COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs:
  SIGNED
***B***
LOCAL COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs:
  SIGNED
REMOTE COMMITS:
 Commit 1:
  Offered htlcs:
  Received htlcs:
  Fee level 2
  SIGNED
